tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc

```diff
@@ -16,6 +16,9 @@ limitations under the License.
 // This file implements logic for lowering HLO/LHLO dialect to Linalg dialect.
 
 #include <numeric>
+#include <set>
```

Comment:
Now using llvm::SmallSetVector<StringRef,4> for slight convenience and *hopefully performance*, since StringSet uses some StringMapEntry when iterating through it